vignettes/new-application.Rmd

```diff
@@ -66,8 +66,14 @@ This is the main body area for applications where charts and figures, tables, et
 
 ### New Application Template Generation
 
-* **Empty application** - ready for the user to drop in their code!
-* **Sample application** - demonstrates framework features in a working application
+* User have two main types of application to generate:
```

Comment:
Reword: 
User can generate two main types of applications

Reply:
done